src/NServiceBus.Hosting.Windows/NServiceBus.Host.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>net472</TargetFramework>
+    <TargetFrameworks>net472;net6.0</TargetFrameworks>
     <DebugType>full</DebugType>
     <RootNamespace>NServiceBus.Hosting.Windows</RootNamespace>
     <SignAssembly>true</SignAssembly>
```

Comment:
There's still a NSB Core beta 1 reference below here where I can't comment

Reply:
I updated the dependency to NSB core beta 2